bin/esm/build.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const esbuild = require( 'esbuild' );
```

Comment:
I've used esbuild because I didn't manage to use webpack to generate esmodules output, apparently the support is there as experimental but I failed to make it work. Also esbuild is very very very fast.